index.html

```diff
@@ -0,0 +1,183 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:ital,wght@0,100;0,300;0,700;1,100&display=swap" rel="stylesheet">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:ital,wght@0,100;1,100&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="index.css">
+    <title>Document</title>
+</head>
+<body>
+<section class="header-title">
+    <nav class="header-text">
+        <a href="#" class="text">Kinyera Amos</a>
+    </nav>
+    <nav class="image-header">
+        <img src="assets/images/Union.png" alt="" >
+    </nav>
+</section>
+<div class="main-title"><h1 class="sub-title title5">Laura Jong</h1></div>
+<div class="main-content">
+    <h1>Software Developer</h1>
+    <p class="large-paragraph">
+        I can help you build a product , feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.
+    </p>
+</div>
+<div class="sub-title">
+    <h3 class="paragraph1">Language</h3>
+    <h3 class="paragraph2">Frameworks</h3>
+</div>
+<div class="sub-title">
+    <p class="experience">JavaScript</p>
+    <p class="experience">Bootstrapt</p>
+</div>
+<div class="sub-title">
+    <p class="experience">Ruby</p>
+    <p class="experience">Ruby on Rails</p>
+</div>
+<div class="sub-title">
+    <p class="experience">Html</p>
+    <p class="experience">RESPec</p>
+</div>
+<div class="sub-title">
+    <p class="experience">Css</p>
+    <p class="experience">CapyBara</p>
+</div>
+<div class="sub-title">
+    <p class="selenium">Selenium</p>
+</div>
+<div class="skill">
+    <p class="inner-head">Skills</p>
+    <div class="inner-title">
+        <p class="experience">CodeKit</p>
+    </div>
+    <div class="inner-title">
+        <p class="experience" >Github</p>
+    </div>
+    <div class="inner-title">
+        <p class="experience">Codepen</p>
+    </div>
+    <div class="inner-title">
+        <p class="experience">Gitlab</p>
+    </div>
+    <div class="inner-title">
+        <p class="experience">Terminal</p>
+    </div>
+</div>
```

Comment:
> I think instead of
> 
> tag i suggest using
> 
> * tag and  tag use  this would be better.

- [ ] Please make it clear on your review what you are trying to suggest. I think it would be better if you say with an example.

Reply:
- [ ] You can also add why semantic tags are important, they help your website to be  easily read by screen readers and in SEO optimization